packages/forms/tsconfig.build.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "./tsconfig.json",
+  "exclude": ["**/pkg/**/*", "**/*.spec.*"],
+  "compilerOptions": {
+    "outDir": "pkg/dist-types"
```

Comment:
why?

Reply:
You mean the `outDir`. It was copied from sibling packages.